framework/components/ecl-tabs/ecl-tabs.twig

```diff
@@ -0,0 +1,20 @@
+<div class="ecl-tabs">
```

Comment:
A question here. When the D8 (potentially) uses the templates from our packages, is it that they can use it better if we provide the information in the templates via context?